app/views/docs/databases.phtml

```diff
@@ -226,16 +225,47 @@ func main() async throws {
     </li>
 </ul>
 
+<h2><a href="/docs/databases#indexes" id="indexes">Indexes</a></h2>
+<p>
+    Databases use indexes to quickly locate data without having to search through every document for matches.
+    To ensure the best performance, Appwrite requires an index for every query.
```

Comment:
This is no longer true, but still worth mentioning that it will improve query performance if you do add them